flarefly/fitter.py

```diff
@@ -931,7 +931,7 @@ def get_signal_over_background(self, idx=0, nsigma=3):
         signal_over_background_err = np.sqrt(signal[1]**2/signal[0]**2 + bkg[1]**2/bkg[0]**2)
         signal_over_background_err *= signal_over_background
 
-        return signal_over_background, signal_over_background_err
+        return float(signal_over_background), float(signal_over_background_err)
```

Comment:
There should not be need since both signal and background returned by `get_signal` and `get_background` now are floats
```suggestion
        return signal_over_background, signal_over_background_err
```

Reply:
Sorry, I did not see your comment. I will change it right away :)